src/features/PersonDetails/AboutPerson/styled.js

```diff
@@ -0,0 +1,101 @@
+import styled from "styled-components";
+
+export const AboutPersonContainer = styled.div`
+  display: grid;
+  gap: 40px;
+  max-width: 1368px;
+  margin: 56px auto;
+  background-color: ${({ theme }) => theme.color.white};
+  padding: 40px;
+  grid-template-columns: auto 1fr;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    margin-top: 0px;
+  }
+`;
+
+export const PersonImage = styled.img`
+  max-width: 500px;
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+  border-radius: 5px;
+  grid-row: span 2;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    max-width: 250px;
+    grid-row: auto;
```

Comment:
What do you think to add more breakpoints for PersonImage?
You could start adding smallLaptop breakpoint (1172px) and then tablet breakpoint etc.